.github/workflows/integration-tests.yml

```diff
@@ -60,7 +61,7 @@ jobs:
       - name: Checkout the repo
         uses: actions/checkout@v2
       - name: Run Tests
-        uses: smartcontractkit/ctf-ci-e2e-action@41848377703354a32d355d3f2f7cb3b9e3049b8c
+        uses: smartcontractkit/ctf-ci-e2e-action@f8fb1000b7978b136097e3e15e7b06182c5399ee
```

Comment:
Which branch is that commit on?

Didn't see it here: https://github.com/smartcontractkit/ctf-ci-e2e-action/commits/main